src/calibrationlib/csv_writer.cpp

```diff
@@ -0,0 +1,114 @@
+#include "csv_writer.hpp"
+
+#include <map>
+#include <string>
+#include <utility>
+
+#include <fstream>
+
+#include <boost/algorithm/string.hpp>
+
+namespace opossum {
+
+CSVWriter::CSVWriter(
+    const std::string file_path,
+    const std::vector<std::string>
+        headers,  // headers for csv file, if the file for file_path already exists, it must have equal headers
```

Comment:
Does the linter actually check your source code folders? Can you add a `TODO: test` and check please? Such a todo should fail in the linter.

Reply:
The linter triggers on TODO: test

Reply:
And did you format and that changed something? Maybe format does not include the correct folders?

Reply:
The formatter works fine too. Actually I think the formatter added the line break after `std::vector<std::string>` because the following comment is too long